api/src/main/java/org/openmrs/api/ConditionService.java

```diff
@@ -62,6 +62,17 @@
 	@Authorized({ PrivilegeConstants.GET_CONDITIONS })
 	List<Condition> getActiveConditions(Patient patient) throws APIException;
 
+	/**
+	 * Gets a condition by its database primary key
```

Comment:
Just say, Gets a condition by id

It may not be the database primary key field and even if it were, you should not expose it to the service client